installer/datafiles/base_omsagent.data

```diff
@@ -93,7 +93,6 @@ MAINTAINER:              'Microsoft Corporation'
 /opt/microsoft/omsagent/plugin/heartbeat_lib.rb;                        source/code/plugins/heartbeat_lib.rb;                  744; root; root
 /opt/microsoft/omsagent/plugin/filter_patch_management.rb;              source/code/plugins/filter_patch_management.rb;        744; root; root
 /opt/microsoft/omsagent/plugin/patch_management_lib.rb;                 source/code/plugins/patch_management_lib.rb;           744; root; root
-/opt/microsoft/omsagent/plugin/filter_linux_update_run_progress.rb;     source/code/plugins/filter_linux_update_run_progress.rb;           744; root; root
 
```

Comment:
Any reason you haven't added a line in base_omsagent.data for your new .conf file? Like so:

/etc/opt/microsoft/omsagent/sysconf/omsagent.d/update_management.conf;      installer/conf/omsagent.d/update_management.conf;